tests/acceptance/features/webUISharingAcceptShares/acceptShares.feature

```diff
@@ -241,6 +241,23 @@ Feature: accept/decline shares coming from internal users
     And as "Brian" folder "from_Alice" should exist inside folder "/Shares/simple-folder" in the server
     And as "Brian" folder "from_Carol" should exist inside folder "/Shares/simple-folder (2)" in the server
 
+  @skipOnOC10
+  Scenario: receive shares with same name from different users, accept one by one
+    Given user "Carol" has been created with default attributes and without skeleton files in the server
+    And user "Carol" has created folder "/simple-folder" in the server
+    And user "Carol" has created folder "/simple-folder/from_Carol" in the server
+    And user "Carol" has shared folder "/simple-folder" with user "Brian" in the server
+    And user "Alice" has created folder "/simple-folder" in the server
+    And user "Alice" has created folder "/simple-folder/from_Alice" in the server
+    And user "Alice" has shared folder "/simple-folder" with user "Brian" in the server
+    And the user has browsed to the shared-with-me page
+    When the user accepts share "simple-folder" offered by user "Alice Hansen" using the webUI
+    Then folder "simple-folder" shared by "Alice Hansen" should be in "Accepted" state on the webUI
+    When the user accepts share "simple-folder" offered by user "Carol King" using the webUI
+    Then folder "simple-folder" shared by "Carol King" should be in "Accepted" state on the webUI
+    And as "Brian" folder "from_Alice" should exist inside folder "/Shares/simple-folder" in the server
+    And as "Brian" folder "from_Carol" should exist inside folder "/Shares/simple-folder" in the server
```

Comment:
Note:  this seems like strange behavior, but if that is the way that oCIS should work then that is what we test.